addOns/zest/src/main/java/org/zaproxy/zap/extension/zest/ExtensionZest.java

```diff
@@ -1507,8 +1508,11 @@ public void startClientRecording(String uri) {
         clientUrlToWindowHandle.clear();
         startRecordingUrl = uri;
         recordingWinId = 0;
-        // And turn off the recording button, at least for now
-        this.getRecordButton().setSelected(false);
```

Comment:
Why? Shouldn't we disable the button still?

Reply:
Oops deleted these by mistake